docs/services.md

```diff
@@ -128,15 +138,15 @@ Accessing a `Service` without a selector works the same as if it had selector.
 The traffic will be routed to endpoints defined by the user (`1.2.3.4:80` in
 this example).
 
-## Portals and service proxies
+## Virtual IPs and service proxies
```

Comment:
Possible ways to relate VIPs and cluster IPs:
* Virtual cluster IPs
* Virtual IPs (aka cluster IPs)
* Cluster-local virtual IPs